src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt

```diff
@@ -130,10 +131,11 @@ class ZipView(path: Path) : ToolPanel("ins 6, flowy") {
                     }
                     add(openIndividually)
 
-                    if (selectedPaths.size == 1) {
+                    val selectedNode = selectedPaths.first().lastPathComponent as PathNode
+
+                    if (selectedPaths.size == 1 && selectedNode.isLeaf) {
```

Comment:
You might be able to express this slightly cleaner with `selectedPaths.singleOrNull()`, but this is totally fine too

Reply:
Hm, definitely not following on this logic. The isLeaf check removes the option from folders.

Reply:
As in, something like this:
```suggestion
                    val selectedNode = selectedPaths.singleOrNull()?.lastPathComponent as? PathNode

                    if (selectedNode?.isLeaf == true) {
```

Not a big deal at all, though.